x-pack/plugins/embeddable_enhanced/public/plugin.ts

```diff
@@ -40,12 +46,30 @@ export interface StartDependencies {
   embeddable: EmbeddableStart;
   uiActionsEnhanced: AdvancedUiActionsStart;
 }
+export interface SerializedReactEmbeddableDynamicActions {
```

Comment:
Can this interface be removed? I doesn't look like its used any where.

Reply:
Done in 343e464a979251cae24177f87eb348c52ab673b0